pkgs/development/python-modules/pypandoc/default.nix

```diff
@@ -1,33 +1,33 @@
-{ stdenv, buildPythonPackage, fetchPypi
-, pip, pandoc, glibcLocales, haskellPackages, texlive }:
+{ stdenv, buildPythonPackage, fetchFromGitHub
+, pandoc, glibcLocales, haskellPackages, texlive }:
 
 buildPythonPackage rec {
-  pname = "pypandoc";
-  version = "1.4";
-
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "e914e6d5f84a76764887e4d909b09d63308725f0cbb5293872c2c92f07c11a5b";
+  pname = "pypandoc-unstable";
+  version = "2018-06-18";
+
+  src = fetchFromGitHub {
+    owner = "bebraw";
+    repo = "pypandoc";
+    rev = "87912f0f17e0a71c1160008df708c876d32e5819";
+    sha256 = "0l6knkxxhmni4lx8hyvbb71svnhza08ivyklqlk5fw637gznc0hx";
   };
 
-  # Fix tests: first requires network access, second is a bug (reported upstream)
+  # Fix tests: requires network access
   preConfigure = ''
     substituteInPlace tests.py --replace "pypandoc.convert(url, 'html')" "'GPL2 license'"
-    substituteInPlace tests.py --replace "pypandoc.convert_file(file_name, lua_file_name)" "'<h1 id=\"title\">title</h1>'"
   '';
 
-  LC_ALL="en_US.UTF-8";
-
-  propagatedBuildInputs = [ pip ];
-
-  buildInputs = [ pandoc texlive.combined.scheme-small haskellPackages.pandoc-citeproc glibcLocales ];
+  propagatedNativeBuildInputs = [
+    pandoc
+    texlive.combined.scheme-small
+    haskellPackages.pandoc-citeproc
+    glibcLocales
```

Comment:
why do these need to be propagated?

Reply:
I copied this over from the other PR, doesn't make sense to me, especially it shouldn't be native.

Reply:
Removed, native, but should be propagated to set up the PATH properly, otherwise tests fail for example. There'd probably be a solution without propagation though I guess.

Reply:
I'll look into it this can definitely be improved.

Reply:
you do not want to propogate binaries like this. If the package shells out to them, then you want to patch the call sights so that the package can work without having to wrap anything

Reply:
as an example:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/pydot/default.nix
https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/pydot/hardcode-graphviz-path.patch